### PR TITLE
build/rtl8721csm: Update config of loadable_apps

### PR DESCRIPTION
**Pull request by jeongarmy**
- Change network settings
 . CONFIG_NET_TCPIP_THREAD_PRIO=105
 . CONFIG_NET_TCP_OVERSIZE=1460
- Change RR interval to 10
 . CONFIG_RR_INTERVAL=10
- Disable debug options

### CODE REVIEW

**Review by sunghan-chang — DISMISSED** [review]

**Comment by sunghan-chang** [comment]
@pillip8282 @an4967 Could you review this change?

File: build/configs/rtl8721csm/loadable_apps/defconfig [hunk 1]
```diff
@@ -672,7 +669,7 @@ CONFIG_NET_MEM_SIZE=81920
 # CONFIG_NET_TCPIP_CORE_LOCKING is not set
 # CONFIG_NET_TCPIP_CORE_LOCKING_INPUT is not set
 CONFIG_NET_TCPIP_THREAD_NAME="LWIP_TCP/IP"
-CONFIG_NET_TCPIP_THREAD_PRIO=100
+CONFIG_NET_TCPIP_THREAD_PRIO=106
```
Comment (sunghan-chang) [inline build/configs/rtl8721csm/loadable_apps/defconfig:672]: This should be 105, not 106.

**Review by jeongchanKim — DISMISSED** [review]
LGTM

**Review by an4967 — APPROVED** [review]
LGTM

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**